integration-testing/rnode_testing/rnode.py

```diff
@@ -176,71 +186,71 @@ def execution():
             process.join()
             raise TimeoutError(cmd, self.timeout)
 
-    def shell_out(self, *cmd, stderr=True):
+    def shell_out(self, *cmd: Union[str, int], stderr=True) -> str:
```

Comment:
Again, we should correct the code so that the type declaration can read: 
```suggestion
    def shell_out(self, *cmd: Union[str], stderr=True) -> str:
```